### PR TITLE
test_cgroup_cpuset is failing intermittently on certain platform

### PR DESCRIPTION
**Pull request by anamikau**
#### Bug/feature Description
* *Bugs: test_cgroup_cpuset is failing intermittently on certain platform*

#### Affected Platform(s)
* *RHEL7*

#### Cause / Analysis / Design
* *Bugs: test fails while comparing the assigned cpus for each job by cgroups. I looked at the mom logs and cgroups hook working okay. There is some other race condition somewhere hence adding these log messages to help debugging*

#### Solution Description
* *Adding more log messages to help debugging the issue in future.*

#### Testing logs/output
* *Please attach your test log output from running the test you added (or from existing tests that cover your changes)
[test_cgroup_cpuset.txt](https://github.com/PBSPro/pbspro/files/2269210/test_cgroup_cpuset.txt)

*

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: test/tests/functional/pbs_cgroups_hook.py [hunk 1]
```diff
@@ -232,6 +232,7 @@ def setUp(self):
     jobnum=${PBS_JOBID%%.*}
     base="$cpuset_base/pbspro.slice/pbspro-${jobnum}.*.slice"
 fi
+echo "cpuset base path is $base"
```
Comment (mike0042) [inline test/tests/functional/pbs_cgroups_hook.py]: This looks like shell syntax, not Python syntax.

**Comment by bayucan** [comment]
@anamikau :The added log messages looked okay, although, you have a Travis issue.

File: test/tests/functional/pbs_cgroups_hook.py (hunk 1 above)
Comment (anamikau) [reply test/tests/functional/pbs_cgroups_hook.py]: @mike0042 this is a shell script because.

**Comment by anamikau** [comment]
Thanks for looking into this @mike0042 and @bayucan. I have fixed pep8 issues reported by Travis.

File: test/tests/functional/pbs_cgroups_hook.py (hunk 1 above)
Comment (mike0042) [reply test/tests/functional/pbs_cgroups_hook.py]: Sorry, I needed more lines for context. I see now.

**Comment by anamikau** [comment]
Thanks @mike0042 and @bayucan .

@mike0042 I have squashed and rebased. Please take this forward as soon as the checks passes.

**Comment by agrawalravi90** [comment]
Hey Anamika, please change the title of your pull request, and the commit message to "test_cgroup_cpuset is failing intermittently on certain platform". The PR title and the commit message should summarize the bug as it affects a user.

**Comment by anamikau** [comment]
Thanks for pointing it out @agrawalravi90 . Updated.

**Comment by mike0042** [comment]
Please rebase your PR so we can get it merged today.

**Comment by anamikau** [comment]
rebased again ... waiting for the checks to pass.

**Pull request merged**